packages/caliper-fabric/lib/fabric.js

```diff
@@ -2279,6 +2280,13 @@ class Fabric extends BlockchainInterface {
      * @async
      */
     async installSmartContract() {
+        // With flow conditioning, this phase is conditionally required
+        if (!this.initPhaseCompleted ) {
```

Comment:
What if both init and install is skipped? Will the users be initialized?

Reply:
The next interaction with a SUT comes at the client spawn phase, which includes the configuration user credentials to interact .. so we are safe there.